README.md

```diff
@@ -13,7 +61,7 @@ DATABASES = {
     }
 }
 ```
-## Make Migrations
+### Make Migrations
  - Make migrations
  ```
  python manage.py makemigrations
```

Comment:
`"python3" manage.py ...` maybe? I don't know..

Reply:
Thank you. That would be better.